copy-index.js

```diff
@@ -0,0 +1,60 @@
+const students = [
+    {
+        name : "Peyton",
+        grade : 47
+    },
+    {
+        name : "Emily",
+        grade : 56
+    },
+    {
+        name : "Claus",
+        grade : 35
+    },
+    {
+        name : "Nelly",
+        grade : 72
+    },
+    {
+        name : "Brandon",
+        grade : 80
+    },
+    {
+        name : "Trent",
+        grade : 79
+    },
+    {
+        name : "Chris",
+        grade : 61
+    },
+    {
+        name : "James",
+        grade : 60
+    },
+    {
+        name : "Alex",
+        grade : 86
+    },
+    {
+        name : "Khloe",
+        grade : 97
+    }
+]
+
+const passingScore = 60;
+
+function getPassingStudents (arr){
+
+    let passingStudents = [];
+
+    for (const obj of arr){
```

Comment:
this just shows you copied the example from MDN without understanding what it represents in your case.

arr should be renamed to students in your case and obj here is an individual student